drivers/sensor/tdk/icp101xx/EmbUtils/InvBool.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2020 TDK Invensense
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+/** @brief    Custom definition for boolean type to avoid compiler disrepenrencies
+ *	@{
+ */
+
+#ifndef _INV_BOOL_H_
+#define _INV_BOOL_H_
+
+typedef int inv_bool_t;
```

Comment:
#include <stdbool.h> instead and use bool? Why do we need to redefine bool values

Reply:
The goal of his PR is to make sure Zephyr driver for ICP101xx sensors is using the official driver from TDK.
This code is part of this official TDK driver and cannot be modified for Zephyr.